prime-dotnet-webapi/Controllers/OrganizationsController.cs

```diff
@@ -110,6 +123,116 @@ public async Task<ActionResult<Organization>> CreateOrganization(CreateOrganizat
             );
         }
 
+        // POST: api/Organizations/claim
+        /// <summary>
+        /// Claim an existing Organization.
+        /// </summary>
+        [HttpPost("claim", Name = nameof(ClaimOrganization))]
+        [ProducesResponseType(typeof(ApiMessageResponse), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
```

Comment:
All of our (Authenticated) endpoints return 403 as well as 401, so the unfortunate bit of additional boilerplate is to add

```C#
[ProducesResponseType(StatusCodes.Status403Forbidden)]
```